django/utils/http.py

```diff
@@ -232,30 +232,31 @@ def urlsafe_base64_decode(s):
 
 def parse_etags(etag_str):
     """
-    Parses a string with one or several etags passed in If-None-Match and
-    If-Match headers by the rules in RFC 2616. Returns a list of etags
-    without surrounding double quotes (") and unescaped from \<CHAR>.
+    Parses a string of ETags given in an If-None-Match or If-Match header as
```

Comment:
When changing docstrings, we are updating them to use PEP 257 verb style: Parse / Return, etc.